packages/thycotic_ss/manifest.yml

```
@@ -0,0 +1,39 @@
+format_version: 2.0.0
+name: thycotic_ss
+title: "Thycotic Secret Server"
+version: 0.0.1
+source:
+  license: "Elastic-2.0"
+description: "Thycotic Secret Server logs"
+type: integration
+categories:
+  - security
+conditions:
+  kibana.version: "^8.4.1"
+  elastic.subscription: "basic"
+screenshots:
+  - src: /img/sample-screenshot.png
+    title: Sample screenshot
+    size: 600x600
+    type: image/png
+icons:
+  - src: /img/thycotic.svg
+    title: Thycotic Logo
+    size: 64x64
+    type: image/svg+xml
+policy_templates:
+  - name: logs
+    title: Secret Server
+    description: Collect Thycotic Secret Server logs from syslog or a file.
+    inputs:
+      - type: udp
+        title: Collect logs from Secret Server via UDP
+        description: Collecting syslog from Secret Server via UDP
+      - type: tcp
+        title: Collect logs from Secret Server via TCP
+        description: Collecting syslog from Secret Server via TCP
+      - type: logfile
+        title: Collect logs from Secret Server via file
+        description: Collecting syslog from Secret Server via file.
+owner:
+  github: elastic/integrations
```

```suggestion
  github: elastic/security-external-integrations
```
Related, can you add a line to the .github/CODEOWNERS file
```
/packages/thycotic_ss @elastic/security-external-integrations
```